thinkhazard/static/less/report.less

```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
+  margin: 5px;
+  background: rgba(255,255,255, 0.5);
+  border-radius: 5px;
+  border: 2px solid rgba(255,255,255,0.75);
+  padding: 5px;
+  font-size: 14px;
```

Comment:
Font-size seem to be the same for all the .btn-xs (overview, download, zoom-out). Let's set the font-size for those buttons globally by overriding bootstrap.